src/mjolnir/valhalla_add_elevation.cc

```diff
@@ -30,90 +34,97 @@ enum Input { CONFIG, TILES };
  * - tiles - stays for the path to a tile file.
  * */
 
-std::pair<std::string, std::vector<std::string>> parse_arguments(int argc, char** argv) {
-  opt::Options options(
-      "help",
-      " Usage: valhalla_add_elevation [options]\n"
-      "valhalla_add_elevation is a tool for loading elevations for a provided tile. "
-      "The service checks if required elevations stored locally if they are not "
-      "it tries to establish connection to the remote storage(based on the information from configuration file)"
-      "and loads required elevations.\n");
-
-  options.add_options()("h,help",
-                        "Print usage")("c,config", "Path to the configuration file.",
-                                       opt::value<
-                                           std::string>())("t,tiles", "Tiles to add elevations to",
-                                                           opt::value<std::vector<std::string>>());
-
-  options.allow_unrecognised_options();
-  std::string config_file;
-  std::vector<std::string> tiles;
+std::deque<GraphId> get_tile_ids(const boost::property_tree::ptree& pt,
```

Comment:
this is just one bigger refactor I felt like doing, to make this exe somewhat consistent with all others.

in a follow up PR I'll actually break out all of this boilerplate arg parsing to a private header